File: javascript/Day_1/Question_09.js
number = 2;

const check = () => {
  if (number % 2 === 0) {
    return `${number} is even number`;
  } else if (number === 1) {
    return `${number} is composite number`;
  } else {
    return `${number} is odd number`;
  }
};

const result = check(number);

console.log(result);

File: python/Day_1/Question_09.py
number = int(input('Number: '))

def convert(number):
    if(number % 2 == 0):
        return f"{number} is even number"
    elif(number % 2 != 0 and number == 1):
        return f"{number} is odd & composite number"
    else:
        return f"{number} is odd number"

result = convert(number)

print(result)
